plugins/nodes/src/js/columns/NodesTableTypeColumn.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import { compareValues } from "#PLUGINS/nodes/src/js/utils/compareValues";
+import Node from "#SRC/js/structs/Node";
+// TODO: DCOS-39079
+// import { IWidthArgs as WidthArgs } from "@dcos/ui-kit/packages/table/components/Column";
+import { IWidthArgs as WidthArgs } from "#PLUGINS/nodes/src/js/types/IWidthArgs";
+import { SortDirection } from "plugins/nodes/src/js/types/SortDirection";
+import { TextCell } from "@dcos/ui-kit";
+
+export function typeRenderer(data: Node): React.ReactNode {
+  const type = data.isPublic() ? "Public" : "Private";
+  return (
+    <TextCell>
+      <span>{type}</span>
+    </TextCell>
+  );
+}
+
+export function typeSorter(data: Node[], sortDirection: SortDirection): Node[] {
+  // current implementation is a rough idea, not sure if it is the best one…
+  const sortedData = data.sort((a, b) =>
+    compareValues(
+      a.isPublic().toString(),
+      b.isPublic().toString(),
+      a.getHostName().toLowerCase(),
+      b.getHostName().toLowerCase()
+    )
+  );
+  return sortDirection === "ASC" ? sortedData : sortedData.reverse();
+}
+
+export function typeSizer(args: WidthArgs): number {
+  return Math.max(100, args.width / args.totalColumns);
```

Comment:
This time we know exactly the size of our content, so I would suggest a more accurate measurement. On mobile it takes up too much space:

![bildschirmfoto 2018-07-25 um 10 15 52](https://user-images.githubusercontent.com/1337046/43188277-04afc770-8ff4-11e8-82e8-2567002051c4.png)

Reply:
I think 60 looks better: 
![bildschirmfoto 2018-07-25 um 10 19 05](https://user-images.githubusercontent.com/1337046/43188367-38ce9996-8ff4-11e8-802b-32b7a644cbb5.png)